docs/src/main/docs/guides/05_Dockerfile.adoc

```diff
@@ -281,13 +287,143 @@ modules to the `jlink` command:
 
 [source,yaml,subs="verbatim,attributes"]
 ----
-RUN ["jlink", "--compress=2", "--strip-debug", "--no-header-files" \ 
+RUN ["jlink", "--compress=2", "--strip-debug", "--no-header-files" \
      "--add-modules", \
      "java.base,java.logging,java.sql,java.desktop,java.management,java.naming,jdk.unsupported", \
      "--output", "/var/tmp/myjre"]
 ----
 
-=== How Big is This Stuff?
+
+== Creating a Docker Image Using Jib
+
+https://github.com/GoogleContainerTools/jib[Jib] is a collection of build tools for
+constructing Docker images for Java applications. Jib allows you to build images
+straight from a Maven (or Gradle) plugin without the need for the `docker`
+command. It also uses a
+https://github.com/GoogleContainerTools/distroless[distroless] base image that
+can result in a smaller final image.
+
+We'll be using the `jib-maven-plugin` in this example. With
+`jib-maven-plugin` you don't need a Dockerfile
+and it creates a final Docker image that follows a set
+of best-practices for packaging Java applications in a container.
+
+Docker is only required if you are going to install the image locally.
```

Comment:
Maybe clarify ... 

1. `docker` (the command) is only required... (put code ticks)

2. The Docker _application_ is only required...

Reply:
Fixed.